core/src/test/java/org/infinispan/jmx/MvccLockManagerMBeanTest.java

```diff
@@ -79,7 +83,12 @@ public void testNumberOfLocksAvailable() throws Exception {
 
       assertAttributeValue("NumberOfLocksHeld", 1);
       assertAttributeValue("NumberOfLocksAvailable", initialAvailable - 1);
-      tm.getTransaction().runCommit(true);
+      try {
+         tm.getTransaction().runCommit(true);
+         fail("RollbackException expected!");
```

Comment:
Hmmm, then I guess I have a problem with `runCommit(true)` always throwing an exception :)

It seems to be only used in tests, so couldn't the tests that use it throw the exception themselves?